src/common/fam_libfabric.cpp

```diff
@@ -1493,10 +1476,13 @@ void fabric_get_quiet(Fam_Context *famCtx) {
     ssize_t ret = 0;
     uint64_t rxLastFailCnt = famCtx->get_num_rx_fail_cnt();
     int timeout_wait_retry_cnt = 0;
-
     rxcnt = famCtx->get_num_rx_ops();
+    rxLastFailCnt = famCtx->get_num_rx_fail_cnt();
+    cout << "Entered fam_get_quiet: rxcnt " << rxcnt
+         << " , rxlastFailCnt: " << rxLastFailCnt << endl;
 
```

Comment:
you can remove the changes to quiet() calls, as currently, we would support multithreaded model only for blocking calls.

Reply:
these cout statements have been deleted